app/src/layouts/projectLayout/projectSidebar/projectSidebar.jsx

```diff
@@ -166,6 +166,7 @@ export const ProjectSidebar = ({ onClickNavBtn }) => {
     shortTitle: `${projectName[0]}${projectName[projectName.length - 1]}`,
     topTitle: `${formatMessage(messages.organization)}: ${organizationName}`,
     bottomTitle: projectName,
+    isProjectLevel: true,
```

Comment:
Let's not rely on boolean here to make it more extendable, I suggest to use smth like `level: 'project'`.